newamericadotorg/assets/react/surveys-homepage/components/TeaserListing.js

```diff
@@ -266,7 +266,7 @@ class TeaserListing extends Component {
               key={`teaser-listing-item-${index}`}
               target="_blank"
             >
-              <div className="col-sm-9">
+              <div className="col-sm-9 col-md-10">
```

Comment:
Need to confirm date does not break on all screen widths

Reply:
Everything seems to maintain continuity above 270px.